config/locales/diaspora/en.yml

```diff
@@ -175,6 +175,7 @@ en:
     zero: "no aspects"
     one: "1 aspect"
     other: "%{count} aspects"
+    inaspects: "in {1} aspects"
```

Comment:
We already have `shared.aspect_dropdown.toggle`. You are changing this with javascript so you might also want to use some appropriate translation in `javascript.en.yml` and check if we add those translations to the mobile website